examples/pidof.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
```

Comment:
Add the license here:

# Copyright (c) 2009, Giampaolo Rodola'. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.